README.rst

```diff
@@ -95,15 +68,29 @@ Steps to install TrinityX
 
     # ansible-galaxy install OndrejHome.pcs-modules-2
 
-9. Start TrinityX installation::
+9. Configure ``hosts`` file to allow ansible to address controllers.
+
+
+    Example for non-HA setup::
+
+        [controllers]
+        controller ansible_host=10.141.255.254
+
+    Example for HA setup::
+
+        [controllers]
+        controller1 ansible_host=10.141.255.254
+        controller2 ansible_host=10.141.255.253
+
+10. Start TrinityX installation::
 
-     # ansible-playbook site.yml |& tee -a install.log
+     # ansible-playbook -i hosts controller.yml |& tee -a install.log
 
    **Note**: If errors are encoutered during the installation process, analyze the error(s) in the output and try to fix it then re-run the installer.
 
-10. Create a default OS image::
+11. Create a default OS image::
 
-    # ansible-playbook image.yml |& tee -a image.log
+    # ansible-playbook compute.yml |& tee -a image.log
```

Comment:
for the future, I'd name it something like _compute-image.yml_ for the sake of clearness